doc/source/groupby.rst

```diff
@@ -1301,11 +1315,15 @@ Piping can also be expressive when you want to deliver a grouped object to some
 arbitrary function, for example:
 
 .. code-block:: python
+   def mean(groupby):
```

Comment:
need a blank line here

Reply:
also here, `.. code-block:: python` requires the `>>>` formatting, and the output, use the `ipython` directive (I know this was wrong before)

Reply:
@jreback Added blank line, see 28eb57f0f731b72e901158161792a0d364c1a661